Providers/Profile.swift

```diff
@@ -436,8 +441,33 @@ open class BrowserProfile: Profile {
     }
 
     lazy var placesDbPath = URL(fileURLWithPath: (try! files.getAndEnsureDirectory()), isDirectory: true).appendingPathComponent("places.db").path
-
-    lazy var places = RustPlaces(databasePath: placesDbPath)
+    lazy var browserDbPath =  URL(fileURLWithPath: (try! self.files.getAndEnsureDirectory())).appendingPathComponent("browser.db").path
+    lazy var places: RustPlaces = RustPlaces(databasePath: self.placesDbPath)
+
+    public func migrateHistoryToPlaces(migrationConfig: HistoryMigrationConfiguration, callback: @escaping (HistoryMigrationResult) -> Void, errCallback: @escaping (Error?) -> Void) {
+        let lastSyncTimestamp = Int64(self.syncManager.lastSyncFinishTime ?? 0)
```

Comment:
note that this probably not the right timestamp, I need to verify which timestamp is

Reply:
Hmm I think this is as good of a timestamp as we can get from my looking around - but happy to change it to something else if anyone knows any better, maybe @lougeniaC64 - not sure if your research into sync-manager revealed a better timestamp.